impl/src/main/java/com/sun/faces/config/manager/tasks/ParseConfigResourceToDOMTask.java

```diff
@@ -109,10 +110,14 @@
      * Constructs a new ParseTask instance
      * </p>
      *
-     * @param servletContext the servlet context.
-     * @param validating whether or not we're validating
-     * @param documentURI a URL to the configuration resource to be parsed
-     * @throws Exception general error
+     * @param servletContext
```

Comment:
I know it's a tiny nit and the rest of the Mojarra code is certainly not compliant, but the Jakarta EE coding standards ask to not indent javadoc. Its same line, or when too large (more than 120 chars) beginning of next line.